src/main/java/org/broadinstitute/hellbender/engine/filters/PlatformUnitReadFilter.java

```diff
@@ -21,7 +21,7 @@
     private static final long serialVersionUID = 1L;
 
     @Argument(fullName = "blackListedLanes",
-            shortName = "blackListedLanes", doc="Keep reads with platform units not on the list",
+            shortName = "blackListedLanes", doc="Keep reads with platform units not on the list.",

```

Comment:
same as previously, arg doc string != filter purpose

Also the explanation could be clearer, right now it's a bit convoluted